gui/static/js/components/controlpanel.js

```diff
@@ -0,0 +1,295 @@
+                {/*<button className="controls" id="removeBot">Remove Bot</button><br/>*/}
+                Power ({this.state.power}): <input id="power" type="range" name="power" min="0" max="100" value={this.state.power} defaultValue="50" onChange={this.handleInputChange}/><br/>
+                <b>Directions:</b><br/>
+                <table>
+                    <tbody>
+                    <tr>
+                        <td className="controlgrid"><button className="btn" id="ccw" onClick={this.sendKV}>turn CCW</button></td>
+                        <td className="controlgrid"><button className="btn" id="fwd" onClick={this.sendKV}>forward</button></td>
+                        <td className="controlgrid"><button className="btn" id="cw" onClick={this.sendKV}>turn CW</button></td>
+                    </tr>
+                    <tr>
+                        <td className="controlgrid"><button className="btn" id="lft" onClick={this.sendKV}>left</button></td>
+                        <td className="controlgrid"><button className="btn btn-danger" id="stop" onClick={this.sendKV}>STOP</button></td>
+                        <td className="controlgrid"><button className="btn" id="rt" onClick={this.sendKV}>right</button></td>
+                    </tr>
+                    <tr>
+                        <td className="controlgrid"><button className="btn btn-success" id="log" onClick={this.sendKV}>log data</button></td>
+                        <td className="controlgrid"><button className="btn" id="bck" onClick={this.sendKV}>backward</button></td>
+                        <td className="controlgrid"></td>
+                    </tr>
+                    <tr>
+                        <td>
+                            Keyboard Controls <br/>
+                            <label className="switch">
+                                <input type="checkbox" id="keyboard-controls"/>
+                                <span className="slider"></span>
+                            </label>
+                        </td>
+                        <td></td>
+                        <td>
+                            Xbox Controls <br/>
+                            <label className="switch">
+                                <input type="checkbox" id="xbox-controls"/>
+                                <span className="slider"></span>
+                            </label>
+                        </td>
+                    </tr>
+                    {/*<tr>*/}
+                        {/*<td><input type="text" id="kv_key" placeholder="Key (e.g. WHEELS)"/></td>*/}
+                        {/*<td><input type="text" id="kv_value" placeholder="Value (e.g. 10,10)"/></td>*/}
+                        {/*<td><button id="sendkv" onClick={this.sendKV} className="btn btn-success">Send KV</button></td>*/}
+                    {/*</tr>*/}
+                    </tbody>
+                </table>
+            </div>
+        )
+    }
+}
+
+// $(document).ready(function() {
```

Comment:
Is this commented out temporarily? If you're planning on replacing code you can probably just delete this.